packages/compiler-cli/src/ngtsc/program.ts

```diff
@@ -430,6 +431,7 @@ export class NgtscProgram implements api.Program {
         // - error TS2531: Object is possibly 'null'.
         // - error TS2339: Property 'value' does not exist on type 'EventTarget'.
         checkTypeOfDomEvents: false,
+        checkTypeOfReferences: true,
```

Comment:
The new flags should all default to `false` even with TTC on

```suggestion
        checkTypeOfReferences: false,
```